dev/.env.docker-compose-native

```
@@ -7,3 +7,4 @@ DJANGO_MINIO_STORAGE_ACCESS_KEY=minioAccessKey
 DJANGO_MINIO_STORAGE_SECRET_KEY=minioSecretKey
 DJANGO_STORAGE_BUCKET_NAME=django-storage
 DJANGO_HOMEPAGE_REDIRECT_URL=http://localhost:8080/
+REDIS_URL=redis://redis:6379
```

Since this is the `-native` file, these should be available on `localhost`.
```suggestion
REDIS_URL=redis://localhost:6379
```

You'll also want to add back in the port forwarding on the redis service in `docker-compose.yml`.

---

Changed in [12b65cb](https://github.com/OpenGeoscience/uvdat/pull/118/commits/12b65cbb9e66200b9bb72f6d0b117118bc640826)